packages/waku/src/lib/plugins/vite-plugin-deploy-aws-lambda.ts

```diff
@@ -8,6 +8,8 @@ import { DIST_PUBLIC } from '../builder/constants.js';
 
 const SERVE_JS = 'serve-aws-lambda.js';
 
+const lambda_streaming = process.env.DEPLOY_AWS_LAMBDA_STREAMING === 'true';
```

Comment:
just naming convention:
```suggestion
const lambdaStreaming = process.env.DEPLOY_AWS_LAMBDA_STREAMING === 'true';
```